core/src/main/java/com/digitalpebble/stormcrawler/bolt/FeedParserBolt.java

```diff
@@ -50,7 +50,6 @@
 import org.xml.sax.InputSource;
 
 /** Extracts URLs from feeds */
-@SuppressWarnings("serial")
```

Comment:
`BaseRichBolt` implements `Serializable`, so should'nt we add a serial version uuid to honor the contract? At least, if we remove the suppression ;)

The same is true for other sub-classes of the `BaseRichBolt` in the project.